component-server-parent/component-server/src/test/java/org/talend/sdk/component/server/front/ComponentResourceImplTest.java

```diff
@@ -219,31 +219,6 @@ void migrateFromStudioWs() {
         assertEquals("true", migrated.get("migrated"));
     }
 
-    @Test
-    void migrateWithEncrypted() {
```

Comment:
Why do we remove this test? I suppose this test still can validate that after the migration the vault value remained the same. Or do I misunderstood the idea of test?

Reply:
effectively, can keep it but adapted.